python/pyspark/sql/tests.py

```diff
@@ -3036,6 +3052,9 @@ def test_toPandas_arrow_toggle(self):
         pdf = df.toPandas()
         self.spark.conf.set("spark.sql.execution.arrow.enable", "true")
         pdf_arrow = df.toPandas()
+        # need to remove timezone for comparison
+        pdf_arrow["7_timestamp_t"] = \
+            pdf_arrow["7_timestamp_t"].apply(lambda ts: ts.tz_localize(None))
```

Comment:
I talked to @gatorsmile and he suggested that we should have another configuration to control the behavior of `df.toPandas()` to handle timezone or not, the default behavior of which is to not handle it as the same as the current behavior.
I'll submit a pr to do it with "without-Arrow" version asap and I'd like you to follow the behavior.